docs/reference-architectures/data/enterprise-bi-sqldw.md

```diff
@@ -0,0 +1,402 @@
+
+**Power BI**. Power BI is a suite of business analytics tools to analyze data for business insights. In this architecture, it queries the semantic model stored in Analysis Services.
+
+**Azure Active Directory** (Azure AD) authenticates users who connect to the Analysis Services server through Power BI.
+
+## Data Pipeline
+ 
+This reference architecture uses the [WorldWideImporters](/sql/sample/world-wide-importers/wide-world-importers-oltp-database) sample database as data source. The data pipeline has the following stages:
+
+1. Export the data from SQL Server to flat files (bcp utility).
+2. Copy the flat files to Azure Blob Storage (AzCopy).
+3. Load the data into SQL Data Warehouse (PolyBase).
+4. Transform the data into a star schema (T-SQL).
+5. Load a semantic model into Analysis Services (SQL Server Data Tools).
+
+![](./images/enterprise-bi-sqldw-pipeline.png)
+ 
+> [!NOTE]
+> For steps 1 &ndash; 3, consider using Redgate Data Platform Studio. Data Platform Studio applies the most appropriate compatibility fixes and optimizations, so it's the quickest way to get started with SQL Data Warehouse. For more information, see [Load data with Redgate Data Platform Studio](/azure/sql-data-warehouse/sql-data-warehouse-load-with-redgate). 
+
+The next sections describe these stages in more detail.
+
+### Export data from SQL Server
+
+The [bcp](/sql/tools/bcp-utility) (bulk copy program) utility is a fast way to create flat text files from SQL tables. In this step, you select the columns that you want to export, but don't transform the data. Any data transformations should happen in SQL Data Warehouse.
+
+**Recommendations**
+
+If possible, schedule data extraction during off-peak hours, to minimize resource contention in the production environment. 
+
+Avoid running bcp on the database server. Instead, run it from another machine. Write the files to a local drive. Ensure that you have sufficient I/O resources to handle the concurrent writes. For best performance, export the files to dedicated fast storage drives.
+
+You can speed up the network transfer by saving the exported data in Gzip compressed format. However, loading compressed files into the warehouse is slower than loading uncompressed files, so there is a tradeoff between faster network transfer versus faster loading. If you decide to use Gzip compression, don't create a single Gzip file. Instead, split the data into multiple compressed files.
+
+### Copy flat files into blob storage
+
+The [AzCopy](/azure/storage/common/storage-use-azcopy) utility is designed for high-performance copying of data to into Azure blob storage.
+
+**Recommendations**
+
+Create the storage account in a region that is closest to where the source data resides. Deploy the storage account and the SQL Data Warehouse instance in the same region. 
```

Comment:
"in a region that is closest to where the source data resides" -> "in a region near the source data's location"?
Needs to be near, but not necessarily the _closest_  region?